packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -126,43 +109,39 @@ export class CoreTracer implements types.Tracer {
    */
   startRootSpan<T>(options: types.TraceOptions, fn: (root: types.Span) => T):
       T {
-    return this.contextManager.runAndReturn((root) => {
-      let traceId;
-      if (options && options.spanContext && options.spanContext.traceId) {
-        traceId = options.spanContext.traceId;
-      } else {
-        // New root span.
-        traceId = uuid.v4().split('-').join('');
-      }
-      const name = options && options.name ? options.name : 'span';
-      const kind =
-          options && options.kind ? options.kind : types.SpanKind.UNSPECIFIED;
-
-      let parentSpanId = '';
-      let traceState;
-      if (options && options.spanContext) {
-        // New child span.
-        parentSpanId = options.spanContext.spanId || '';
-        traceState = options.spanContext.traceState;
+    let traceId;
+    if (options && options.spanContext && options.spanContext.traceId) {
```

Comment:
Can we just do `options = options || {name: 'span'};` at the start and remove falsiness check on `options` (multiple places below)?

Reply:
actually, we could do just:

`function (options = { name: 'span' })`